scheduler/server/load_based_sched_alg.go

```diff
@@ -733,6 +733,8 @@ func (lbs *LoadBasedAlg) setClassLoadPercents(classLoadPercents map[string]int32
 		log.Errorf("LoadBalanced scheduling %%'s don't add up to 100, normalizing them")
 		lbs.normalizePercents(pctTotal)
 		log.Errorf("LoadBalanced scheduling class percents have been changed to %v", lbs.config.classLoadPercents)
+	} else {
+		log.Infof("set classLoadPercents to %v", lbs.config.classLoadPercents)
```

Comment:
nit, but prefer we log this info on one line before the "if pctTotal != 100" check, so we don't have to have an else block.

Reply:
done